thirdparty/glib/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ set(gen_arm_config "${gen_arm_config}ac_cv_func_posix_getgrgid_r=no\\n")
 set(gen_arm_config "${gen_arm_config}ac_cv_func_getpwuid_r=no\\n")
 set(gen_arm_config "${gen_arm_config}ac_cv_func_getgrgid_r=no\\n")
 set(gen_arm_config "${gen_arm_config}ac_cv_func_timegm=no\\n")
-set(gen_arm_config sh -c "printf \"${gen_arm_config}\" > arm_cache.conf")
+set(gen_arm_config "printf \"${gen_arm_config}\" > arm_cache.conf")
```

Comment:
Huh, for some reason I thought that was a shell builtin but apparently not.

```
$ which printf
/usr/bin/printf
```

Reply:
It's both ;).

Reply:
I see… well, in any case I don't have to go waving my hands saying it won't work without sh -c. :-P